### PR TITLE
Update zk_login.md to suggest using Base64 instead of BigInt

### PR DESCRIPTION
**Pull request by mskd12**
## Description 

Describe the changes or additions included in this PR.

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

**Review by joyqvq — COMMENTED** [review]
why changing the format too?

**Comment by mskd12** [comment]
Note that we are not disallowing BigInt - both will work.

One of the reasons to allow Base64 is space-efficiency. I also got the sense from Jordan that devs are typically more comfortable with Base64.

**Comment by joyqvq** [comment]
> Note that we are not disallowing BigInt - both will work.
> 
> One of the reasons to allow Base64 is space-efficiency. I also got the sense from Jordan that devs are typically more comfortable with Base64.

can we keep both examples then? 

im concerned that this will cause confusion bc few places in the doc we uses bigint.

File: doc/src/build/zk_login.md [hunk 1]
```diff
@@ -234,6 +234,8 @@ Because generating a ZK proof can be resource-intensive and potentially slow on
 
 Here's an example request and response for the Mysten Labs-maintained proving service. If you wish to use the Mysten ran ZK Proving Service, please contact us for whitelisting your registered client ID. Only valid JWT token authenticated with whitelisted client IDs are accepted.
 
+Note that either BigInt or Base64 encoding can be used for the three fields: `extendedEphemeralPublicKey`, `jwtRandomness` and `salt`. The example below shows two sample requests: the first uses BigInt for the three fields and the second uses Base64.
```
Comment (ronny-mysten) [inline doc/src/build/zk_login.md:237]:
```suggestion
You can use BigInt or Base64 encoding for `extendedEphemeralPublicKey`, `jwtRandomness`, and `salt`. The following examples show two sample requests with the first using BigInt encoding and the second using Base64.
```

**Review by ronny-mysten — COMMENTED** [review]
Content looks good. Leaving for @joyqvq to approve or req changes. Thanks, both!

**Review by joyqvq — APPROVED** [review]

**Pull request merged**